### ISSUE TITLE
How many routers can a framework have

### ISSUE THREAD

**Issue by mbeckenbach** [angular/angular#9160]
Hi guys, 

i love ng1 and feel like i will love ng2. But it feels like everytime I play arround with ng2 there is a new router. Could you please avoid creating a new router version 4, 5, 6, and 7. It's getting hard to follow. :-)

By the way: releasing at least some basic docs on the newest new router would very helpful...

Max

**Comment by ericmartinezr** [comment]
http://angularjs.blogspot.cl/2016/06/improvements-coming-for-routing-in.html
